ranch/utils.py

```diff
@@ -680,7 +683,7 @@ def get_non_near_connectors(all_cc_link_gdf, num_connectors_per_centroid, zone_i
         centroid = zone_cc_gdf.c_point.iloc[0]
 
         # if the zone has less than 4 cc, keep all
-        if len(zone_cc_gdf) <= num_connectors_per_centroid:
+        if len(zone_cc_gdf) <= 4:
```

Comment:
Would it make more sense to change the value to num_connectors_per_centroid than hardcoding?